internals/overlord/servstate/manager_test.go

```diff
@@ -1758,14 +1799,31 @@ func (r testRestarter) HandleRestart(t restart.RestartType) {
 	close(r.ch)
 }
 
-func (s *S) logBufferString() string {
+// readClearLogBuffer reads and clears the current log buffer. If you need
+// to poll for a specific stdout/stderr message, this is not suitable.
+func (s *S) readClearLogBuffer() string {
```

Comment:
Naming: let's call this `readAndClearLogBuffer` for clarity -- now it looks like it's reading the "clear log buffer".

Reply:
Done.